plugins/techdocs/api-report.md

```diff
@@ -465,7 +464,6 @@ export class TechDocsStorageClient implements TechDocsStorageApi_2 {
   constructor(options: {
     configApi: Config;
     discoveryApi: DiscoveryApi;
-    identityApi: IdentityApi;
```

Comment:
Thinking that we should mark this as `@deprecated` and make it optional and just skip using it rather than breaking change. It would only affect people who provide a custom factory to the app for creating a `techdocsApiRef` but it's still technically breaking.

Reply:
I'm not 100% sure the way I marked it as deprecated is fine (couldn't find a similar example). Can you check?

Reply:
I did the comment, but somehow I can't make it show up in the generated api-report.md.

any hints are welcome